kettle/make_json.py

```diff
@@ -176,11 +183,10 @@ def get_metadata():
                 if not isinstance(value, str):
                     # the schema specifies a string value. force it!
                     metadata[key] = json.dumps(value)
-        if not metadata:
-            return None
-        return [{'key': k, 'value': v} for k, v in sorted(metadata.items())]
+            metapairs = [{'key': k, 'value': v} for k, v in sorted(metadata.items())]
+        return metapairs
 
-    metadata = get_metadata()
+    metadata = get_metadata(build)
```

Comment:
not sure I understand why this refactor?

Reply:
So I didn't like doing this but I want to access the metadata `repos` key. I have access to all key-value pairs outside of this internal method but I would have to iterate over them because I no longer have a direct map to the `repos` value. The quickest solution I saw was to edit the map object while handling the metadata.